Tools/autotest/arduplane.py

```diff
@@ -1327,6 +1327,46 @@ def airspeed_autocal(self):
         self.wait_waypoint(7, num_wp-1, timeout=500)
         self.wait_disarmed(timeout=120)
 
+    def deadreckoning_main(self, disable_airspeed_sensor=False):
+        # wind is from the West:
+        self.set_parameter("SIM_WIND_DIR", 270)
+        # light winds:
+        self.set_parameter("SIM_WIND_SPD", 10)
+        if disable_airspeed_sensor:
```

Comment:
When disable_airspeed_sensor is false, SIM_WIND_SP is unchanged, and therefor an unknown value. Would "SIM_WIND_SPD = disable_airspeed_sensor" be better?

Reply:
Uh.  I'm pretty sure we add simulated windspeed regardless of whether we're disabling the airspeed sensor?